lolliplotServer.py

```diff
@@ -5,89 +5,97 @@
 from bokeh.models import LinearColorMapper, ColorBar
 from bokeh.models import palettes
 from bokeh.layouts import widgetbox, column, layout
-from bokeh.models.widgets import Slider, Panel, Tabs, TextInput
+from bokeh.models.widgets import Slider, Panel, Tabs, TextInput, PreText
 from bokeh.models import HoverTool, CustomJS
 
 import numpy as np
 import dataProcessing
 
+import sys
```

Comment:
Is this actually used? I didn't spot it if so.

Reply:
It isn't as is, but I need it when I want to print out debug output and I keep forgetting it, so I just put it there... 😬